tests/functional/spec/features/section_summary/section_summary.spec.js

```diff
@@ -1,89 +1,88 @@
       expect(browser.getUrl()).to.contain(PropertyDetailsSummaryPage.pageName);
     });
 
-    it('When I select edit from Final Summary and change an answer that doesn\'t affect completeness, Then I should be taken to the Section Summary', function() {
-      $(FinalSummaryPage.summaryShowAllButton()).click();
-      $(FinalSummaryPage.insuranceAddressAnswerEdit()).click();
-      $(InsuranceAddressPage.answer()).setValue('Test Address');
+    it("When I select edit from Final Summary and change an answer that doesn't affect completeness, Then I should be taken to the Section Summary", function() {
+      $(QuestionnaireSummaryPage.summaryShowAllButton()).click();
+      $(QuestionnaireSummaryPage.insuranceAddressAnswerEdit()).click();
+      $(InsuranceAddressPage.answer()).setValue("Test Address");
       $(InsuranceAddressPage.submit()).click();
       expect(browser.getUrl()).to.contain(PropertyDetailsSummaryPage.pageName);
     });
 
-    it('When I select edit from Final Summary and change an answer that affects completeness, Then I should be stepped through the section', function() {
-      $(FinalSummaryPage.summaryShowAllButton()).click();
-      $(FinalSummaryPage.insuranceTypeAnswerEdit()).click();
+    it("When I select edit from Final Summary and change an answer that affects completeness, Then I should be stepped through the section", function() {
+      $(QuestionnaireSummaryPage.summaryShowAllButton()).click();
+      $(QuestionnaireSummaryPage.insuranceTypeAnswerEdit()).click();
       $(InsuranceTypePage.contents()).click();
       $(InsuranceTypePage.submit()).click();
       expect(browser.getUrl()).to.contain(InsuranceAddressPage.pageName);
       $(InsuranceAddressPage.submit()).click();
       expect(browser.getUrl()).to.contain(AddressDurationPage.pageName);
     });
 
-    it('When I change an answer, Then the final summary should display the updated value', function() {
-      $(FinalSummaryPage.summaryShowAllButton()).click();
-      expect($(FinalSummaryPage.insuranceAddressAnswer()).getText()).to.contain('No answer provided');
-      $(FinalSummaryPage.insuranceAddressAnswerEdit()).click();
+    it("When I change an answer, Then the final summary should display the updated value", function() {
+      $(QuestionnaireSummaryPage.summaryShowAllButton()).click();
+      expect($(QuestionnaireSummaryPage.insuranceAddressAnswer()).getText()).to.contain("No answer provided");
+      $(QuestionnaireSummaryPage.insuranceAddressAnswerEdit()).click();
       expect(browser.getUrl()).to.contain(InsuranceAddressPage.pageName);
-      $(InsuranceAddressPage.answer()).setValue('Test Address');
+      $(InsuranceAddressPage.answer()).setValue("Test Address");
       $(InsuranceAddressPage.submit()).click();
-      expect($(FinalSummaryPage.insuranceAddressAnswer()).getText()).to.contain('Test Address');
+      expect($(QuestionnaireSummaryPage.insuranceAddressAnswer()).getText()).to.contain("Test Address");
     });
   });
 });
```

Comment:
Any reason why all the quotes have been changed? This has been causing codacy to fail for a while now.

Reply:
Probably my IDE settings - I'll revert these soon; I hadn't had time to look at the codacy failure yet.